packages/create-widget.template.minimal-react.v2/templates/src/main.tsx

```diff
@@ -1,14 +1,15 @@
 import "@blueprintjs/core/lib/css/blueprint.css";
+import "./main.css";
```

Comment:
This file was never being included and it resets the body margin to zero:
https://github.com/palantir/osdk-ts/blob/main/packages/create-widget.template.minimal-react.v2/templates/src/main.css

I updated the widget's container CSS to put the margin back to keep the visuals the same as before